packages/containerd-1.7/containerd-1.7.spec

```diff
@@ -23,6 +23,7 @@ Source3: containerd-config-toml_basic
 Source4: containerd-config-toml_k8s_nvidia_containerd_sock
```

Comment:
For the lack of a better place to post this, there is a typo in this commit message `s/sectinos/sections/`